src/website/src/pages/ec/standardised-template/page-header/docs/code.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Showcase
+order: 2
+---
+
+import { Playground, Html } from '@ecl/website-components';
+
+import {
+  pageHeaderTitle,
+  pageHeaderMetaTitle,
+  pageHeaderMetaTitleDescription,
+} from '../demo';
+
+## Title
+
+<Playground
+  system="ec"
+  selectedKind="components-page-headers-page-header-standardised"
```

Comment:
i noticed this only when reviewing the page header core, i think it's better to rename the story and have the path to it like this: components-page-headers-standardised, like in v2